pom.xml

```diff
@@ -1,339 +1,386 @@
-  <modelVersion>4.0.0</modelVersion>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.joyent.manta</groupId>
-  <artifactId>java-manta</artifactId>
-  <version>1.6.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>9</version>
+    </parent>
 
-  <name>java-manta</name>
-  <url>https://github.com/joyent/java-manta</url>
-  <licenses>
-    <license>
-      <name>The MIT License (MIT)</name>
-      <url>http://opensource.org/licenses/MIT</url>
-      <distribution>repo</distribution>
-      <comments>Copyright (c) 2013 Joyent</comments>
-    </license>
-  </licenses>
-  <description>Java Manta SDK.</description>
+    <groupId>com.joyent.manta</groupId>
+    <artifactId>java-manta</artifactId>
+    <version>1.6.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
 
-  <organization>
-    <name>Joyent</name>
-    <url>joyent.com</url>
-  </organization>
-  <inceptionYear>2012</inceptionYear>
+    <name>java-manta</name>
+    <description>Java Manta SDK.</description>
+    <url>https://github.com/joyent/java-manta</url>
+    <inceptionYear>2012</inceptionYear>
+    <organization>
+        <name>Joyent</name>
+        <url>joyent.com</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>The MIT License (MIT)</name>
+            <url>http://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+            <comments>Copyright (c) 2015 Joyent</comments>
+        </license>
+    </licenses>
 
-  <developers>
-    <developer>
-      <email>yjxiao@gmail.com</email>
-      <name>Yunong Xiao</name>
-      <organization>Joyent</organization>
-      <organizationUrl>joyent.com</organizationUrl>
-      <url>yunong.ca</url>
-      <id>yunong</id>
-    </developer>
-    <developer>
-      <email>elijah@zupancic.name</email>
-      <name>Elijah Zuoancic</name>
-      <organization>Joyent</organization>
-      <organizationUrl>joyent.com</organizationUrl>
-    </developer>
-  </developers>
+    <developers>
+        <developer>
+            <email>yjxiao@gmail.com</email>
+            <name>Yunong Xiao</name>
+            <organization>Joyent</organization>
+            <organizationUrl>joyent.com</organizationUrl>
+            <url>yunong.ca</url>
+            <id>yunong</id>
+        </developer>
+        <developer>
+            <email>elijah@zupancic.name</email>
+            <name>Elijah Zuoancic</name>
+            <organization>Joyent</organization>
+            <organizationUrl>joyent.com</organizationUrl>
+        </developer>
+    </developers>
```

Comment:
Oh, yes.  We should also probably move Yunong from developers section to a contributors section.  I believe the convention is that developers actively communicate and field questions to the community about the project, whereas contributors are people who should not be contracts directly.  If he's no longer actively involved he should be moved to a contributors section.